History.markdown

```diff
@@ -1,16 +1,16 @@
 ## HEAD
 
-  * Release 1.3.0 ### -minor (#58)
-  * Rubocop 0.55 (#61)
-
 ### Minor Enhancements
 
-  * Drop support for Ruby 2.2.x (#60)
+  * Bump Rubocop to 0.55 (#61)
+  * Drop support for Ruby 2.2.x (#61)
+  * Relax version constraint on ActiveSupport (#60)
 
 ## 1.3.0 / 2018-03-14
 
 ### Development Fixes
 
+  * Release 1.3.0 (#58)
```

Comment:
idk if the release needs to be listed in the history at all (dev fix sounds especially weird), but no big deal :+1:

Reply:
You can remove the entry directly on `master` if its a nit.. :wink: